apps/vite/src/pages/Calendar2.tsx

```
@@ -1,58 +1,104 @@
-      date: '१२',
-      day: 'सोम',
-      title: 'रमा एकादशी व्रत',
-      daysLeft: 8,
-      fullDate: 'कार्तिक १२, २०८१ | Oct 28, 2024',
-    },
+  const { BSYear, BSMonth } = useParams()
+  const [view, setView] = useState<"calendar" | "event">("calendar")
+
+  const validYearAndMonth = useMemo(() => {
+    if (!BSYear || !BSMonth) return new NepaliDate()
+    const year = parseInt(BSYear)
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)